TaskLayer/SearchTask/SearchParameters.cs

```diff
@@ -25,7 +25,7 @@ public SearchParameters()
             KeepAllUniprotMods = true;
             MassDiffAcceptorType = MassDiffAcceptorType.OneMM;
             MaxFragmentSize = 30000.0;
-            WriteMzId = true;
+            WriteMzId = false;
```

Comment:
why this change? it doesn't take much extra time to create the .mzID but it is really annoying if you wanted an mzID and forgot to check the box

Reply:
for me its not a time thing, its a space thing. both drives on my computer where i store data are alwasy full and i'm alwasy deleting stuff. my assumption is that a majority of users will never need mzid for anything. so its superfluous. they can tick the box when needed. I will change it back if this is really important to you.

Reply:
it is very important to me.. right now mzid is the only "standard" output that we write (pepxml for xl task also I think) so if anybody wants to output our results into other programs (skyline) it's really important. the space constraints thing is something to consider but with @zrolfs change you can specify your own defaults, so you can disable mzid and save that as your default if you wish

Reply:
reverted